### DISCUSSION TITLE
Capstone - Command Prompt tidak dapat 'activate' environment

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#42] [Q&A] [answered]
![Capture1](https://user-images.githubusercontent.com/95233522/162384177-ba7b6111-47c1-4956-947a-198f3b4e6ad1.JPG)
![Capture2](https://user-images.githubusercontent.com/95233522/162384186-14f18800-16ff-4ffa-ba29-a08398c1171b.JPG)

Mohon pencerahannya. 
Apakah sepertinya karena salah folder, harus ditaruh dibuka yang di "C:\Users\husad\Capstone - Webscrap\web_scraping-main"
Sementara file saya simpan di "C:\Syh\DS\Algorit.ma\Algoritma Academy\Classroom Cohort 18\1. Materi Data Analytics Specializations\Capstone Project - Webscrapping\web_scraping-main\web_scraping-main"
Apa begitu?

**Comment by onlyphantom** [comment]
It looks like you either haven't install `conda` or you have it installed under a virtual environment? 

> Apakah sepertinya karena salah folder, harus ditaruh dibuka yang di "C:\Users\husad\Capstone - Webscrap\web_scraping-main"
Sementara file saya simpan di "C:\Syh\DS\Algorit.ma\Algoritma Academy\Classroom Cohort 18\1. Materi Data Analytics Specializations\Capstone Project - Webscrapping\web_scraping-main\web_scraping-main"

Nggak. If you install `conda` on your system, the directory that you are in does not matter. You can activate conda from anywhere as long as it's installed (and symlinked) correctly.

**Reply by Hertzspr** [reply]
![Capture3](https://user-images.githubusercontent.com/95233522/162409307-38739b94-4b50-4acc-bf57-070ee6e141ad.JPG)
I have Conda installed already, and I set webscrap_hus environment for the capstone project.

**Reply by Hertzspr** [reply]
Should I reinstall conda? Would it not tamper with what already available?

**Comment by triarm** [comment]
Bapak dapat mencoba mengakses anacondanya dengan menekan tombol panah yang berada di samping tanda + kemudian pilih Command Prompt. Setelah itu seharusnya terminal akan membuka anaconda promptnya
![162384177-ba7b6111-47c1-4956-947a-198f3b4e6ad1_LI](https://user-images.githubusercontent.com/20942472/162412744-fb1c71d2-121e-480e-9b61-f84126214518.jpg)

**Reply by Hertzspr** [reply]
Nggak bisa.

**Comment by onlyphantom** [comment, marked as answer]
This [link](https://exerror.com/conda-command-is-not-recognized-on-windows-10/#:~:text=Windows%2010%20Error%20%3F-,To%20Solve%20Conda%20command%20is%20not%20recognized%20on%20Windows%2010,And%20Then%20Add%20New%20Path.) suggests:

- Open Anaconda Prompt and type this command: `where conda` 
- Press windows key and type Open Advanced System Settings. 
- Now, Click on Environment Variables. 
- Edit Path variable. And Then Add New Path. In my case `C:\Users\ssc\Anaconda3\Scripts; C:\Users\ssc\Anaconda3; C:\Users\ssc\Anaconda3\Library\bin;` Just add above 3 in PATH variable. (Change this to what you find from `where conda`)
- Open Command Prompt and Check Versions And type `conda install anaconda-navigator` in cmd then press `y`

**Reply by Hertzspr** [reply]
![Capture4](https://user-images.githubusercontent.com/95233522/162430704-3f9c2811-5de9-4b30-bbe1-30c49364a734.JPG)
Unfortunately it still fails

**Reply by onlyphantom** [reply]
Can you also try this suggestion below:

> Open the Anaconda Prompt program (via the Windows start menu) and run the command `conda init bash`. This only needs to be done once. After that, your shell will be configured to use conda going forward.

**Reply by Hertzspr** [reply]
Thank you for the suggestion, I just tried that, but the VSCode cmd terminal still doesn't work in the way intended unless I launched it via Conda Navigator and specified environtment channel. By default, it doesn't work.

**Comment by Hertzspr** [comment]
I tried to launch VSCode via Conda Navigator, and selected the virtual environment "webscrap_hus", and it seemed to be working. But I still don't understand the underlying mechanism why does it fail, why does it succeed?

![Berhasil](https://user-images.githubusercontent.com/95233522/162431510-b1421e8c-77d9-452f-9ba9-f26cee011d46.JPG)
![Solusi - Berhasil](https://user-images.githubusercontent.com/95233522/162431523-0e8462ef-3573-4cd8-b098-e8369f67728f.JPG)
